### PR TITLE
Improves Setting DateFormat

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
The parameter menu does not indicate what the date format actually stands for. It lacks a universal chronological format. #8249

**Details of Changes**
1) Add Universal format (2022-04-07 20:00:00).
2) Add formatted date examples in settings menu (current format and selectable formats).
3) Fix spelling in localized text code (SystemTimeStye).
4) The code that generates the date in the correct format is no longer duplicated in many places, only in DateTimeExtensions.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
DateFormat

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@cinqmilleans can you revert the auto generated changes to the string files? To make code reviews easier, we only modify the `en-us` file. We'll update the other string files in a separate pull request.

**Comment by cinqmilleans** [comment]
@yair100 It's done.

File: src/Files.Uwp/Views/SettingsPages/Preferences.xaml [hunk 1]
```diff
@@ -51,15 +51,27 @@
...
+                    x:Name="DateFormatChooser" Grid.Column="1"
                     AutomationProperties.Name="{helpers:ResourceString Name=DateFormat}"
                     ItemsSource="{Binding DateFormats}"
-                    SelectedIndex="{Binding SelectedDateFormatIndex, Mode=TwoWay}" />
+                    SelectedIndex="{Binding SelectedDateFormatIndex, Mode=TwoWay}">
+                    <ComboBox.ItemTemplate>
+                        <DataTemplate x:DataType="settingsviewmodels:DateFormatItem">
+                            <StackPanel Orientation="Vertical">
+                                <TextBlock Text="{x:Bind Label}" />
+                                <TextBlock Margin="16,0,0,0" FontWeight="Light" FontSize="12"
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Preferences.xaml:67]: What are your thoughts on using a smaller margin?

Comment (cinqmilleans) [reply src/Files.Uwp/Views/SettingsPages/Preferences.xaml:67]: We can reduce it.  I would do that tomorrow.

**Review by lukeblevins — COMMENTED** [review]
I like this idea. However, the indentation of the added lines seems redundant with the boosted typography above them. We should probably start each line of text from the edge of the ComboBoxItems like normally done.

File: src/Files.Uwp/Views/SettingsPages/Preferences.xaml (hunk 1 above)
Comment (cinqmilleans) [reply src/Files.Uwp/Views/SettingsPages/Preferences.xaml]: It's done.

**Comment by cinqmilleans** [comment]
@yair100 @duke7553 I finally removed the margin. It's pretty.
DateFormatNoMargin

**Review by lukeblevins — APPROVED** [review]

**Review by yair100 — APPROVED** [review]
LGTM

**Comment by yaichenbaum** [comment]
@mdtauk this looks good! I think a single date setting is enough but we can take inspiration from this for a future improvement.

**Pull request merged**